app/gzac/src/main/resources/config/pbac/document-definition.permission.json

```diff
@@ -1,6 +1,11 @@
 {
     "changesetId": "document-definitions-v1",
     "permissions": [
+        {
+            "resourceType": "com.ritense.document.domain.impl.JsonSchemaDocumentDefinition",
+            "action": "export",
```

Comment:
@marijnritense what do you think, should this be `export` or `export-instance` or something?